wcivf/apps/elections/helpers.py

```diff
@@ -81,19 +81,20 @@ def get_data(self, election_id):
         self.ee_cache[election_id] = None
         return None
 
-    def iter_recently_modified_election_ids(self):
+    def iter_recently_modified_election_ids(self, group_type=None):
+        modified_date = timezone.datetime.now().date() - timezone.timedelta(days=20)
```

Comment:
Why is this change (to now minus 20 days vs minus 1 hour) necessary? How often will this be run?

Reply:
I changed this for testing purposes, I wanted to be sure to something was returned to confirme it's working as hoped. I plan to change it back.